packages/OreIdProfile/README.md

```diff
@@ -0,0 +1,144 @@
+# ORE ID React Library
```

Comment:
Looks like this was just copied from the root README.md. Please update with specific instructions for this component.

Reply:
Done